### PR TITLE
Find Duplicate in Array in CPP

### PR DESCRIPTION
**Pull request by shahina-bano**
I have added the code for issue #777 . Kindly review it @tarun26091999 @rudrakshi99

### CODE REVIEW

File: Code/C++/repeated_element.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    {
+        if (x.second > 1)
+        { //printing the numbers whose frequency is more than 1 i.e which is repeated
+            cout << x.first << " ";
+        }
+    }
+}
+
+int main()
+{
```
Comment (sayantani11) [inline Code/C++/repeated_element.cpp:37]: Where is the input statement?

Comment (sayantani11) [reply Code/C++/repeated_element.cpp:37]: Shouldn't be after int main()

**Review by sayantani11 — CHANGES_REQUESTED** [review]

**Comment by shahina-bano** [comment]
@sayantani11 input is in the function
and what shouldn't be after int main? can you please clarify more.

**Comment by sayantani11** [comment]
I know but the conventional way somewhere makes it better for a layman

**Comment by shahina-bano** [comment]
> I know but the conventional way somewhere makes it better for a layman

I think that the way in which I have written the code is quite understandable even for a layman too. If a mentor would ask me to make this change, then I'll do it :)

**Review by sayantani11 — APPROVED** [review]

**Comment by sayantani11** [comment]
Open source is a community where we try to help each other to get better with contributions.....I did suggest changes to few of my participant mates and they took it sportingly and it's similar with me
Anyways your files are **FAULTY** , check before pushing.... It conflicts with main branch

**Comment by sayantani11** [comment]
Thank you

**Comment by shahina-bano** [comment]
> Open source is a community where we try to help each other to get better with contributions.....I did suggest changes to few of my participant mates and they took it sportingly and it's similar with me
> Anyway your files are **FAULTY** , check before pushing.... It conflicts with main branch

thank you for your suggestion, I am not opposing it though :). I'll consider this next time onwards :) @sayantani11

File: Code/C++/repeated_element.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Repeated elements are: 3 2 
+
+Test Case 2:
+Input -
+10
+1 1 1 2 2 2 3 4 4 5
+output -
+Repeated elements are: 1 2 4 
+
+Time Complexity: O(size), for traversing the elements
```
Comment (raksha009) [inline Code/C++/repeated_element.cpp:57]: Specify size

File: Code/C++/repeated_element.cpp [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+5
+1 2 3 2 3
+output -
+Repeated elements are: 3 2 
+
+Test Case 2:
+Input -
+10
+1 1 1 2 2 2 3 4 4 5
+output -
```
Comment (raksha009) [inline Code/C++/repeated_element.cpp:54]:
```suggestion
Output :

```

File: Code/C++/repeated_element.cpp [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+int main()
+{
+    print_repeat_numbers(); //calling for the function in the main
+    return 0;
+}
+
+/*
+Test Case 1:
```
Comment (raksha009) [inline Code/C++/repeated_element.cpp:43]: Follow 
Test Cases
Input : 
Output :

File: Code/C++/repeated_element.cpp [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+                   Print all the elements whose frequency is greater than 1.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+void print_repeat_numbers()
+{
+    int size;
+    cin >> size; //taking the size as input from user
```
Comment (raksha009) [inline Code/C++/repeated_element.cpp:12]: Add comments above the line throughout yr code.

**Review by raksha009 — CHANGES_REQUESTED** [review]

File: Array/readme.md [hunk 6]
```diff
@@ -25,7 +25,8 @@ To create an array, you need to known the length (or size) of the array in advan
 ## Questions :
 
  * Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp) 
- * Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp)  
+ * Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp) 
+ * Find duplicate in array ----> [C++](/Code/C++/repeated_element.cpp)
```
Comment (raksha009) [inline Array/readme.md:29]: Plz have a suitable file name related to problem statement.

File: Code/C++/repeated_element.cpp [hunk 7]
```diff
@@ -0,0 +1,64 @@
+/*Find Duplicate in array
+Problem Statement: Given an array with size "size", the elements inside the array can be any number of times.
```
Comment (raksha009) [inline Code/C++/repeated_element.cpp:2]:
```suggestion
Problem Statement: Given an array with size "size", the elements inside the array can be present any number of times.

```

File: Code/C++/repeated_element.cpp [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+    return 0;
+}
+
+/*
+Test Cases
+1.
+Input :
+5
+1 2 3 2 3
+output :
```
Comment (raksha009) [inline Code/C++/repeated_element.cpp:52]:
```suggestion
Output :

```

File: Code/C++/repeated_element.cpp [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+Repeated elements are: 3 2 
+
+2.
+Input :
+10
+1 1 1 2 2 2 3 4 4 5
+output :
```
Comment (raksha009) [inline Code/C++/repeated_element.cpp:59]: O capital

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Review by raksha009 — APPROVED** [review]

**Pull request merged**